src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -1138,6 +1134,20 @@ def __del__(self):
             # "Cannot invoke RPC on closed channel!"
             pass
 
+    # Currently getting an error whenever I'm calling add_on_command_attribute_changed on self or parent
+    # def _get_cached_attr(self, attrib: str) -> Any:
+    #     cached_val = self.cached_attrs.get(attrib)
+    #     if cached_val is None:
+    #         cached_val = self._get_remote_attr(attrib)
+    #         parent = PyStateContainer(self.service, self.rules, self.path[0:-1])
+    #         parent.add_on_command_attribute_changed(
```

Comment:
Getting an error from server with either
```
eventrequest {
  rules: "meshing"
  commandAttributeChangedEventRequest {
    path: "/ImportGeometry:ImportGeometry2"
    command: "ImportGeometry"
    attribute: "FileName/isActive"
  }
}
```
or
```
eventrequest {
  rules: "meshing"
  commandAttributeChangedEventRequest {
    command: "ImportGeometry"
    attribute: "FileName/isActive"
  }
}
```
requests.
We have unittest with command-level attributes.

Reply:
command argument attributes won't be cached as they depends on command state in client, not entirely depends on server state